lib/src/question.dart

```diff
@@ -0,0 +1,15 @@
+class Question {
+  final QuestionType questionType;
+  final String label;
+  final bool showsInfoIcon;
+  dynamic answer;
```

Comment:
Acho que podia ser logo um info no lugar de showsInfoIcon. Como os nosso info são apenas imagens, podia ser uma String referenciando o link.
answer devia ser instância da classe Answer, não?

Reply:
É que precisaria definir direito o answer.
Como ainda não tem é melhor deixar livre.

Eu acho q o info vale mais a pena só existir assim e no controller ter uma função que avisa quando ele for clicado. Ai quem ta chamando decide o q fazer. A gente pode exibir o nosso dialog com a imagem mas em outro projeto chamar uma route por exemplo.

Reply:
também pensei nisso, ai pode ser uma Functino onTapInfo então, mas ajeita isso depois